lib/Steps/ChooseIntegration.ts

```diff
@@ -12,14 +12,19 @@ import { Cordova } from './Integrations/Cordova';
 import { Electron } from './Integrations/Electron';
 import { hasPackageInstalled } from '../../src/utils/package-json';
 import { dim } from '../Helper/Logging';
+import { readFileSync } from 'node:fs';
 
-let projectPackage: any = {};
+let projectPackage: Record<string, unknown> = {};
 
 try {
   // If we run directly in setup-wizard
-  projectPackage = require('../../package.json');
+  projectPackage = JSON.parse(
+    readFileSync('../../package.json', 'utf-8'),
```

Comment:
Funny story about these: `require` is more fault tolerant than `JSON.parse`. Bun decided to permit comments in `package.json` files. Whatever drove them to make this decision is beyond me but here we are. `JSON.parse` will crash on Json with comments. 

I opened #561 a while ago but closed it because it never came up in reality. I'm fine with using JSON.parse for the moment and revisit when people actually report crashes. 

Gotta love the JS ecosystem

Reply:
It is tricky to make dynamic requires work with the static binaries. It is not impossible but is a bit more explicit (you need to use `node:module` and do `makeRequire` with an explicit starting path).

If we get complaints, I can switch to that.